tests/online_migration/sle12_online_migration/pre_migration.pm

```diff
@@ -0,0 +1,72 @@
+# SLE12 online migration tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "consoletest";
+use strict;
+use testapi;
+
+sub set_scc_proxy_url() {
+    if (!get_var("SCC_PROXY_URL")) {
+        my ($scc_proxy_url, $addon, $uc_addon);
+
+        if (get_var("FLAVOR") =~ /(Server|Desktop)/) {
+            $scc_proxy_url = "http://" . $1 . '-' . get_var("BUILD");
+        }
+
+        if (get_var("SCC_ADDONS")) {
+            for $addon (split(/,/, get_var("SCC_ADDONS", ''))) {
+                $uc_addon = uc $addon;
+                $scc_proxy_url .= ".$uc_addon-" . get_var("BUILD_$uc_addon");
+            }
+        }
+
+        $scc_proxy_url .= ".proxy.scc.suse.de";
```

Comment:
openQA already provides proxy scc url for each build, that's how it's designed to work with Proxy SCC